packages/orbit-components/src/SegmentedSwitch/SwitchSegment/index.tsx

```diff
@@ -40,8 +40,8 @@ const SwitchSegment = ({
       <div
         className={cx(
           "orbit-switch-segment-text",
-          "peer-checked:text-form-element-label-filled-foreground peer-checked:outline-blue-normal peer-checked:z-10 peer-checked:!rounded-[5px] peer-checked:outline peer-checked:outline-2 ",
-          "peer-focus:text-form-element-label-filled-foreground peer-focus:outline-blue-normal peer-focus:z-10 peer-focus:!rounded-[5px] peer-focus:outline peer-focus:outline-2 ",
+          "peer-checked:text-blue-normal peer-checked:outline-blue-normal peer-checked:z-10 peer-checked:!rounded-[5px] peer-checked:outline peer-checked:outline-2 ",
+          "peer-[:focus:not(:checked)]:text-form-element-label-filled-foreground peer-focus:outline-blue-normal peer-focus:z-10 peer-focus:!rounded-[5px] peer-focus:outline peer-focus:outline-2 ",
```

Comment:
```suggestion
          "peer-focus:peer-[:not(:checked)]:text-form-element-label-filled-foreground peer-focus:outline-blue-normal peer-focus:z-10 peer-focus:!rounded-[5px] peer-focus:outline peer-focus:outline-2 ",
```

I played around with the selector and this works. At least the focus selector is "tailwind native" and not arbitrary. In the end, the compiled CSS will look the same, it's just a matter of avoiding a bigger arbitrary selector. WDYT?

Reply:
Yes, it makes more sense, thank you!

Adapted.